src/applications/combined-debt-portal/combined/sass/partials/_balance-card.scss

```diff
@@ -0,0 +1,25 @@
+@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
+
+.card-balance {
+  font-family: "Bitter";
+  font-size: 24px;
```

Comment:
This font size doesn't seem to exist in the design system: https://design.va.gov/foundation/utilities/font-size